### ISSUE TITLE
Option to keep primary window size if it is the only window open

### ISSUE THREAD

**Issue by gprimosch** [ianyh/Amethyst#83]
Currently, when you close all windows but the primary window, the primary window expands to take up the entire screen. It would be good to have an option that would leave the primary column unchanged, even if it was the only window open. 

My motivation is that resizing windows is often undesirable since it rearranges certain elements. I would prefer to have a fixed window size for my primary, and let the secondary column be blank.

[Trello Card](https://trello.com/c/C37KUMt8/29-amethyst-option-to-keep-primary-window-size-if-it-is-the-only-window-open)

**Comment by Bellavene** [comment]
Yeah, that would be great as a built-in option. Because "Restore layout's" is jaming amethyst's plist file sometimes.
Also you anyway resize the main pane everytime, depending on your monitor's aspect ratio.

**Comment by ianyh** [comment]
This is possible with custom layouts now.
